public/static/docs/command-reference/import.md

```diff
@@ -130,7 +129,7 @@ Several of the values above are pulled from the original stage file
 subfields under `repo` are used to save the origin and version of the
 dependency, respectively.
 
-## Example: Fixed revisions & re-importing
+## Example: Fixed revisions and updating to different revision
```

Comment:
```suggestion
## Example: Fixed revisions
```

Reply:
Action sound better ... probably even Importing and updating fixed revisions